examples/cosmicshear.yaml

```diff
@@ -1,17 +1,36 @@
 parameters:
   Omega_k: 0.0
-  Omega_c: 0.27
+  Omega_c: [0.25, 0.27, 0.32]  # [min, start, max]
   Omega_b: 0.045
   h: 0.67
   n_s: 0.96
-  sigma8: 0.8
+  A_s: [2.0e-9, 2.1e-9, 2.2e-9]
   w0: -1.0
   wa: 0.0
 
   # lens bin zero
   src0_delta_z: 0.0
   src1_delta_z: 0.0
 
+
+sampler:
```

Comment:
We should call this block `cosmosis` to indicate that it is for interfacing with cosmosis?

Reply:
Isn't that a bit less clear?  Do the users really care that they're using cosmosis to sample?  Don't they just want to sample stuff?

Reply:
yes It is clearer. I want people to be able to distinguish between what firecrown is processing and what cosmosis is processing. This will be hard in general, but we should do it wherever we can. It will help in debugging.

Reply:
Changed.